front/app/modules/commercial/user_custom_fields/utils/data/index.ts

```diff
@@ -0,0 +1,5 @@
+export { default as binBirthyear } from './binBirthyear';
+export { default as join } from './join';
```

Comment:
Might be a personal preference but I perceive this more as a `merge` that a `join`. I don't mind leaving it like this, feels clear enough. I think it's just because I associate `join` with the array method.

Reply:
Yeah in a JS context that makes sense, I was thinking of `join` in a SQL or data manipulation sense like in [tidyverse](https://dplyr.tidyverse.org/reference/mutate-joins.html) or [pandas](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.join.html)